frontend/src/components/PostCategories.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const PostCategories = ({ categories }) => {
+  return categories.map((category, index) => (
+    <p key={index}>{category}</p>
```

Comment:
I think that you should pass category id as a key cause categories may switch between indexes (if for example new is added - category with index 0 then will have 1) and react will be forced to rerender cause it may think that new element appeared.

Reply:
That's why we should avoid the index as a key :-)